src/robot_interface/models/sensor/sensor.py

```diff
@@ -0,0 +1,26 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class Sensor:
+    name: str
+
+
+@dataclass
+class ThermalCamera(Sensor):
+    name: str = "thermal_camera"
+
+
+@dataclass
+class OpticalCamera(Sensor):
+    name: str = "optical_camera"
+
+
+@dataclass
+class Microphone(Sensor):
+    name: str = "microphone"
+
+
+@dataclass
+class LIDAR(Sensor):
+    name: str = "lidar"
```

Comment:
I'm still not seeing the real need of adding the sensor class. If it's because we would like to hide Steps from the robot itself we have introduced a new thing called Sensor which the robot needs to understand.

It is more intuitive than Step seen from the robots POV, however we can solve that by renaming Step maybe?

One reason could be if we intend to support multiple sensors of the same type, however I'm not certain how we would like ISAR to manage possible different image sensors for different robots.

Reply:
Think I agree with Arnt here, because we dont have any aditional information on the sensors other than the name string. I think we can wait creating sensor classes until we need more information on the sensors.